### PR TITLE
added precision parameter to to_s() and pretty(), fixes #21

### LINKED ISSUE #21: Support selection of precision
It should be possible to chose the precision of the output. 

Currently it will always print with 2 decimals. 
It would be a parameter to the `pretty` method, mostly before the `to_s` is already hardcoded with a `unit = 'B'`.

It could also be a format_as method.

### PR DESCRIPTION
**Pull request by lauer**
Please report back what you like?

I have borrow from the ActionView::NumberHelper.number_with_precision()  from the Rails project.

### CODE REVIEW

File: lib/filesize.rb [hunk 1]
```diff
@@ -61,16 +64,18 @@ def to(unit = 'B')
   # @param (see #to_f)
   # @return [String] Same as {#to_f}, but as a string, with the unit appended.
   # @see #to_f
-  def to_s(unit = 'B')
-    "%.2f %s" % [to(unit).to_f.to_s, unit]
+  def to_s(unit = 'B', args = {})
+    precision ||= (args[:precision] || PRECISION)
```
Comment (dominikh) [inline lib/filesize.rb]: ||= should be =, right?

Comment (lauer) [reply lib/filesize.rb]: Yes, you are right. Easy fix

**Comment by lauer** [comment]
@dominikh Do you accept this change or do you have other comments?

**Comment by lauer** [comment]
@dominikh A little reminder for review of the new patch

**Pull request merged**